DigitalLearningSolutions.Web/Controllers/LearningContentController.cs

```diff
@@ -1,35 +1,43 @@
-        private const string ReasonableAdjustmentFlagBrand = "ReasonableAdjustmentFlag";
-        private const string ReasonableAdjustmentFlagTitle = "Reasonable adjustment flag";
-        private const string TerminologyAndClassificationsDeliveryServiceBrand = "TerminologyandClassificationsDeliveryService";
-        private const string TerminologyAndClassificationsDeliveryServiceTitle = "Terminology and classifications delivery service";
-        
-        public IActionResult ItSkillsPathway()
-        {
-            var model = new LearningContentViewModel(ItSkillsPathwayBrand, ItSkillsPathwayTitle);
-            return View("Index", model);
-        }
-        
-        public IActionResult ReasonableAdjustmentFlag()
-        {
-            var model = new LearningContentViewModel(ReasonableAdjustmentFlagBrand, ReasonableAdjustmentFlagTitle);
-            return View("Index", model);
-        }
-        
-        public IActionResult TerminologyAndClassificationsDeliveryService()
-        {
-            var model = new LearningContentViewModel(TerminologyAndClassificationsDeliveryServiceBrand, TerminologyAndClassificationsDeliveryServiceTitle);
-            return View("Index", model);
-        }
-    }
-}
+﻿namespace DigitalLearningSolutions.Web.Controllers
+{
+    using DigitalLearningSolutions.Web.Attributes;
+    using DigitalLearningSolutions.Web.ViewModels.LearningContent;
+    using Microsoft.AspNetCore.Mvc;
+
+    [RedirectDelegateOnlyToLearningPortal]
+    public class LearningContentController : Controller
+    {
+        private const string ItSkillsPathwayBrand = "ITSkillsPathway";
+        private const string ItSkillsPathwayTitle = "IT skills pathway";
+        private const string ReasonableAdjustmentFlagBrand = "ReasonableAdjustmentFlag";
+        private const string ReasonableAdjustmentFlagTitle = "Reasonable adjustment flag";
+
+        private const string TerminologyAndClassificationsDeliveryServiceBrand =
+            "TerminologyandClassificationsDeliveryService";
+
+        private const string TerminologyAndClassificationsDeliveryServiceTitle =
+            "Terminology and classifications delivery service";
+
+        public IActionResult ItSkillsPathway()
+        {
+            var model = new LearningContentViewModel(ItSkillsPathwayBrand, ItSkillsPathwayTitle);
+            return View("Index", model);
+        }
+
+        public IActionResult ReasonableAdjustmentFlag()
+        {
+            var model = new LearningContentViewModel(ReasonableAdjustmentFlagBrand, ReasonableAdjustmentFlagTitle);
+            return View("Index", model);
+        }
+
+        public IActionResult TerminologyAndClassificationsDeliveryService()
+        {
+            var model = new LearningContentViewModel(
+                TerminologyAndClassificationsDeliveryServiceBrand,
+                TerminologyAndClassificationsDeliveryServiceTitle,
+                true

```

Comment:
You can label parameter values when calling a method.
It's generally so that you can specify parameters in whatever order you want, but it's also helpful when you're passing literals, because you don't have variable names to tell the reader what the value means.
Especially with boolean parameters, it's quite hard to work out what significance they have without reading the method declaration.

Could you add one on the `true`? It should end up looking like `longName: true` (or whatever new name you give `longName`).

Reply:
Added the label titleIsLong, it gets removed when I run the formatter though, is that alright?

Reply:
Intriguing. When you say formatter, do you mean the autoformat function? Are you using VS or Rider?

Reply:
Yeah in Rider. It says `Inconsistent argument style: redundant name identifier` when I hover over the label.
But without the label the IDE does bring up a label automatically:
![image](https://user-images.githubusercontent.com/70278044/133752161-d56311f7-fed8-4dd8-bb5f-370d49719f56.png)

Reply:
I think it's useful to have the explicit label. We should change your configuration so that it doesn't auto-remove it, but let's look at that when I'm back on Tuesday.